### PR TITLE
[zh-cn] resync apiserver-config.v1beta1 reference

### PR DESCRIPTION
**Pull request by mengjiao-liu**
### Description

The file `content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md` is outdated and needs to resync.

See [the preview page](https://deploy-preview-48700--kubernetes-io-main-staging.netlify.app/zh-cn/docs/reference/config-api/apiserver-config.v1beta1/).

### CODE REVIEW

File: content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md [hunk 1]
```diff
@@ -142,106 +248,1085 @@ TracingConfiguration 为跟踪客户端提供版本化的配置信息。
...
+</td>
+<td>
+   <p>
+   <!--
+   Type refers to the type of the authorizer
+&quot;Webhook&quot; is supported in the generic API server
+Other API servers may support additional authorizer
+types like Node, RBAC, ABAC, etc.
+   -->
+   Type 指的是鉴权器的类型。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md:386]:
```suggestion
   type 指的是鉴权器的类型。
```

File: content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md [hunk 2]
```diff
@@ -142,106 +248,1085 @@ TracingConfiguration 为跟踪客户端提供版本化的配置信息。
...
+<td>
+   <p>
+   <!--
+   Name used to describe the webhook
+This is explicitly used in monitoring machinery for metrics
+Note: Names must be DNS1123 labels like <code>myauthorizername</code> or
+subdomains like <code>myauthorizer.example.domain</code>
+Required, with no default
+   -->
+   用于描述 webhook 的名称。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md:404]:
```suggestion
   name 用于描述 webhook 的名称。
```

File: content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md [hunk 3]
```diff
@@ -142,106 +248,1085 @@ TracingConfiguration 为跟踪客户端提供版本化的配置信息。
...
+<a href="#apiserver-k8s-io-v1beta1-WebhookConfiguration"><code>WebhookConfiguration</code></a>
+</td>
+<td>
+   <p>
+   <!--
+   Webhook defines the configuration for a Webhook authorizer
+Must be defined when Type=Webhook
+Must not be defined when Type!=Webhook
+   -->
+   Webhook 定义了 Webhook 鉴权器的配置。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md:422]:
```suggestion
   webhook 定义 Webhook 鉴权器的配置。
```

File: content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md [hunk 4]
```diff
@@ -142,106 +248,1085 @@ TracingConfiguration 为跟踪客户端提供版本化的配置信息。
...
+username.expression or extra[<em>].valueExpression or claimValidationRules[</em>].expression.
+An example claim validation rule expression that matches the validation automatically
+applied when username.claim is set to 'email' is 'claims.?email_verified.orValue(true)'.
+   -->
+   username 表示用户名属性的一个选项。
+   声明的值必须是单一字符串。
+   与 --oidc-username-claim 和 --oidc-username-prefix 标志相同。
+   如果设置了 username.expression，则表达式必须生成一个字符串值。
+   如果 username.expression 使用 'claims.email'，则必须在 username.expression 或 extra[<em>].valueExpression 或 claimValidationRules[</em>].expression 中使用 'claims.email_verified'。
+   一个示例声明验证规则表达式，当 username.claim 设置为 'email' 时，匹配自动应用的验证规则是 'claims.?email_verified.orValue(true)'。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md:475]:
```suggestion
   如果 username.expression 使用 'claims.email'，则必须在 username.expression 或
   extra[<em>].valueExpression 或 claimValidationRules[</em>].expression 中使用 'claims.email_verified'。
   这里有一个声明验证规则表达式的示例，当 username.claim 设置为 'email' 时与自动应用的验证所匹配：
   'claims.?email_verified.orValue(true)'。
```

File: content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md [hunk 5]
```diff
@@ -471,3 +1556,343 @@ UDSTransport 设置通过 UDS 连接 konnectivity 服务器时需要的信息。
...
+<a href="https://pkg.go.dev/k8s.io/apimachinery/pkg/apis/meta/v1#Duration"><code>meta/v1.Duration</code></a>
+</td>
+<td>
+   <!--
+   <p>The duration to cache 'unauthorized' responses from the webhook
+authorizer.
+Same as setting <code>--authorization-webhook-cache-unauthorized-ttl</code> flag
+Default: 30s</p>
+   -->
+   <p>缓存来自 Webhook 鉴权组件的“未授权”响应的持续时间。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md:1675]:
```suggestion
   <p>对来自 Webhook 鉴权组件的“unauthorized”响应的缓存时长。
```

File: content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md [hunk 6]
```diff
@@ -471,3 +1556,343 @@ UDSTransport 设置通过 UDS 连接 konnectivity 服务器时需要的信息。
...
+</td>
+<td>
+   <p>
+   <!--
+   MatchConditionSubjectAccessReviewVersion specifies the SubjectAccessReview
+version the CEL expressions are evaluated against
+Valid values: v1
+Required, no default value
+   -->
+   MatchConditionSubjectAccessReviewVersion 指定评估 CEL 表达式的 SubjectAccessReview 版本
```
[the remark this answers is not in the document]
Comment (tengqm) [reply content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md:1725]:
```suggestion
   MatchConditionSubjectAccessReviewVersion 指定对 CEL 表达式求值时使用的 SubjectAccessReview 版本
```

File: content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md [hunk 7]
```diff
@@ -471,3 +1556,343 @@ UDSTransport 设置通过 UDS 连接 konnectivity 服务器时需要的信息。
...
+</td>
+<td>
+   <p>
+   <!--
+   Controls the authorization decision when a webhook request fails to
+complete or returns a malformed response or errors evaluating
+matchConditions.
+Valid values:
+   -->
+   当 Webhook 请求无法完成或返回格式错误的响应或评估 matchConditions 时出现错误时，控制鉴权决定。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md:1742]:
```suggestion
   控制当 Webhook 请求无法完成或返回格式错误的响应或计算 matchConditions 出现错误时的鉴权决定。
```

File: content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md [hunk 8]
```diff
@@ -471,3 +1556,343 @@ UDSTransport 设置通过 UDS 连接 konnectivity 服务器时需要的信息。
...
+   有效值：
+   </p>
+<ul>
+<!--
+<li>NoOpinion: continue to subsequent authorizers to see if one of
+them allows the request</li>
+<li>Deny: reject the request without consulting subsequent authorizers
+Required, with no default.</li>
+-->
+<li>NoOpinion：继续联系后续鉴权组件，看其中是否有人允许该请求</li>
```
Comment (tengqm) [inline content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md:1752]:
```suggestion
<li>NoOpinion：继续执行后续鉴权组件，看其中是否有组件允许该请求</li>
```

File: content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md [hunk 9]
```diff
@@ -471,3 +1556,343 @@ UDSTransport 设置通过 UDS 连接 konnectivity 服务器时需要的信息。
...
+<li>Deny：拒绝请求而不咨询后续鉴权组件，必填，没有默认值。</li>
```
Comment (tengqm) [inline content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md:1753]:
```suggestion
<li>Deny：拒绝请求而不考虑后续鉴权组件。</li>
```

File: content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md [hunk 10]
```diff
@@ -471,3 +1556,343 @@ UDSTransport 设置通过 UDS 连接 konnectivity 服务器时需要的信息。
...
+</ul>
+</td>
```
Comment (tengqm) [inline content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md:1755]:
```suggestion
必填，没有默认值。
</td>
```

File: content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md [hunk 11]
```diff
@@ -471,3 +1556,343 @@ UDSTransport 设置通过 UDS 连接 konnectivity 服务器时需要的信息。
...
+<td>
+<!--
+   <p>matchConditions is a list of conditions that must be met for a request to be sent to this
+webhook. An empty list of matchConditions matches all requests.
+There are a maximum of 64 match conditions allowed.</p>
+<p>The exact matching logic is (in order):</p>
+-->
+   <p>matchConditions 是将请求发送到此 Webhook 必须满足的条件列表。matchConditions 为空列表表示匹配所有请求。
+   最多允许 64 个匹配条件。 An empty list of matchConditions matches all requests.
+There are a maximum of 64 match conditions allowed.</p>
```
Comment (tengqm) [inline content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md:1781]:
```suggestion
   最多允许 64 个匹配条件。</p>
```

File: content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md [hunk 12]
```diff
@@ -471,3 +1556,343 @@ UDSTransport 设置通过 UDS 连接 konnectivity 服务器时需要的信息。
...
+<tr><td><code>kubeConfigFile</code> <B><!--[Required]-->[必需]</B><br/>
+<code>string</code>
+</td>
+<td>
+   <p>
+   <!--
+   Path to KubeConfigFile for connection info
+Required, if connectionInfo.Type is KubeConfig
+   -->
+   连接信息的 KubeConfig 文件的路径
```
Comment (tengqm) [inline content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md:1854]:
```suggestion
   包含连接信息的 KubeConfig 文件的路径。
```

File: content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md [hunk 13]
```diff
@@ -471,3 +1556,343 @@ UDSTransport 设置通过 UDS 连接 konnectivity 服务器时需要的信息。
...
+   如果 connectionInfo.Type 是 KubeConfig，则为必需项
```
Comment (tengqm) [inline content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md:1855]:
```suggestion
   如果 connectionInfo.type 是 KubeConfig，则为必需项。
```

File: content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md [hunk 14]
```diff
@@ -471,3 +1556,343 @@ UDSTransport 设置通过 UDS 连接 konnectivity 服务器时需要的信息。
...
+</td>
+<td>
+<!--
+   <p>expression represents the expression which will be evaluated by CEL. Must evaluate to bool.
+CEL expressions have access to the contents of the SubjectAccessReview in v1 version.
+If version specified by subjectAccessReviewVersion in the request variable is v1beta1,
+the contents would be converted to the v1 version before evaluating the CEL expression.</p>
+<p>Documentation on CEL: https://kubernetes.io/docs/reference/using-api/cel/</p>
+-->
+   <p>表达式表示将由 CEL 评估的表达式。必须评估为布尔值。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md:1890]:
```suggestion
   <p>表达式表示将由 CEL 求值的表达式。求值结果必须为布尔值。
```

File: content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md [hunk 15]
```diff
@@ -471,3 +1556,343 @@ UDSTransport 设置通过 UDS 连接 konnectivity 服务器时需要的信息。
...
+CEL 表达式可以访问 v1 版本中的 SubjectAccessReview 的内容。
+如果请求变量中 subjectAccessReviewVersion 指定的版本是 v1beta1，
+在评估 CEL 表达式之前，内容将转换为 v1 版本。</p>
```
Comment (tengqm) [inline content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md:1893]:
```suggestion
在计算 CEL 表达式之前，内容将被转换为 v1 版本。</p>
```

File: content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md [hunk 16]
```diff
@@ -111,6 +87,136 @@ TracingConfiguration 为 OpenTelemetry 跟踪客户端提供版本化的配置
...
+      jwt 是一个身份验证器列表，用于使用符合 JWT 的令牌对 Kubernetes 用户进行身份验证。身份验证器将尝试解析原始 ID 令牌，验证其是否由配置的发行者签名。用于验证签名的公钥是通过 OIDC 发现从发行者的公共端点获取的。对于传入的令牌，将按照此列表中指定的顺序尝试每个 JWT 身份验证器。但是请注意，其他身份验证器可能会在 JWT 身份验证器之前或之后运行。JWT 身份验证器相对于其他身份验证器的具体位置在不同版本中既未定义也不稳定。由于每个 JWT 身份验证器必须具有唯一的发行者 URL，因此最多只有一个 JWT 身份验证器会尝试对令牌进行加密验证。</p>
+<p>
+<!--The minimum valid JWT payload must contain the following claims:
+-->
+最小有效 JWT 负载必须包含以下声明：
+{
+&quot;iss&quot;: &quot;https://issuer.example.com&quot;,
+&quot;aud&quot;: [&quot;audience&quot;],
+&quot;exp&quot;: 1234567890,
+&quot;<!-- raw HTML omitted -->&quot;: &quot;username&quot;
```
[the remark this answers is not in the document]
Comment (mengjiao-liu) [reply content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md]: 好的，已修改。

File: content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md [hunk 17]
```diff
@@ -142,106 +248,1085 @@ TracingConfiguration 为跟踪客户端提供版本化的配置信息。
...
+对于声明，如果在传统标志方法中未设置 --oidc-username-claim，
+请在身份验证配置中配置 username.claim=&quot;sub&quot;。
+对于前缀：
+(1) --oidc-username-prefix="-", 未添加前缀到用户名。要实现相同的行为，请在身份验证配置中设置
+username.prefix=&quot;&quot;
+(2) --oidc-username-prefix=&quot;&quot; 并且 --oidc-username-claim != &quot;email&quot;，
+前缀为 &quot;&lt;--oidc-issuer-url 的值&gt;#&quot;。要实现相同的行为，请在身份验证配置中设置
+username.prefix=&quot;#&quot;。
+(3) --oidc-username-prefix=&quot;。要实现相同的行为，请在身份验证配置中设置
+username.prefix=&quot;&quot;。
```
[the remark this answers is not in the document]
Comment (mengjiao-liu) [reply content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md]: 好的，这部分已重新修改。

File: content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md [hunk 18]
```diff
@@ -142,106 +248,1085 @@ TracingConfiguration 为跟踪客户端提供版本化的配置信息。
...
+<!--
+if is_admin claim is present and true, extra attribute - admin: [&quot;true&quot;]
+-->
+如果 is_admin 声明存在且为 true，则额外属性 - admin: [&quot;true&quot;]
+</li>
+<li>
+<!--
+if is_admin claim is present and false or is_admin claim is not present, no extra attribute will be added
+-->
+如果 is_admin 声明存在且为 false 或 is_admin 声明不存在，则不会添加额外属性
```
[the remark this answers is not in the document]
Comment (mengjiao-liu) [reply content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md:616]: 好的，这部分已修改。

File: content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md [hunk 19]
```diff
@@ -142,106 +248,1085 @@ TracingConfiguration 为跟踪客户端提供版本化的配置信息。
...
+   <!--
+   url points to the issuer URL in a format https://url or https://url/path.
+This must match the &quot;iss&quot; claim in the presented JWT, and the issuer returned from discovery.
+Same value as the --oidc-issuer-url flag.
+Discovery information is fetched from &quot;{url}/.well-known/openid-configuration&quot; unless overridden by discoveryURL.
+Required to be unique across all JWT authenticators.
+Note that egress selection configuration is not used for this network connection.
+   -->
+   url 指向颁发者 URL，格式为 https://url 或 https://url/path。
+   此 URL 必须与所提供的 JWT 中的 &quot;iss&quot; 声明以及从发现中返回的颁发者匹配。
```
[the remark this answers is not in the document]
Comment (mengjiao-liu) [reply content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md:1019]: 好的，已统一为“颁发者”。

**Comment by mengjiao-liu** [comment]
All comments addressed. Thank @windsonsea and @tengqm for the pieces of advice.

File: content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md [hunk 20]
```diff
@@ -142,106 +259,1081 @@ TracingConfiguration 为跟踪客户端提供版本化的配置信息。
...
+<ul>
+<li>'claims' is a map of claim names to claim values.
+For example, a variable named 'sub' can be accessed as 'claims.sub'.
+Nested claims can be accessed using dot notation, e.g. 'claims.foo.bar'.</li>
+</ul>
+<p>Documentation on CEL: https://kubernetes.io/docs/reference/using-api/cel/</p>
+<p>Mutually exclusive with claim and prefix.</p>
+-->
+   <p>expression 表示将由 CEL 评估的表达式。</p>
+<p>CEL 表达式可以访问令牌声明的内容，这些内容组成 CEL 变量：</p>
```
Comment (tengqm) [inline content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md:1323]:
```suggestion
<p>CEL 表达式可以访问令牌声明的内容，这些内容被组织成 CEL 变量：</p>
```

File: content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md [hunk 21]
```diff
@@ -471,3 +1563,343 @@ UDSTransport 设置通过 UDS 连接 konnectivity 服务器时需要的信息。
...
+   <!--
+   <p>The API version of the authorization.k8s.io SubjectAccessReview to
+send to and expect from the webhook.
+Same as setting <code>--authorization-webhook-version</code> flag
+Valid values: v1beta1, v1
+Required, no default value</p>
+   -->
+   <p>要发送到 Webhook 并期望从 Webhook 获得的 authorization.k8s.io SubjectAccessReview 的 API 版本。
+与设置 <code>--authorization-webhook-version</code> 标志相同。
+有效值：v1beta1、v1
```
Comment (tengqm) [inline content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md:1717]:
```suggestion
有效值：v1beta1、v1。
```

File: content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md [hunk 22]
```diff
@@ -471,3 +1563,343 @@ UDSTransport 设置通过 UDS 连接 konnectivity 服务器时需要的信息。
...
+<td>
+   <p>
+   <!--
+   MatchConditionSubjectAccessReviewVersion specifies the SubjectAccessReview
+version the CEL expressions are evaluated against
+Valid values: v1
+Required, no default value
+   -->
+   matchConditionSubjectAccessReviewVersion 指定对 CEL 表达式求值时使用的 SubjectAccessReview 版本
+   有效值：v1
```
Comment (tengqm) [inline content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md:1733]:
```suggestion
   有效值：v1。
```

File: content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md [hunk 23]
```diff
@@ -471,3 +1563,343 @@ UDSTransport 设置通过 UDS 连接 konnectivity 服务器时需要的信息。
...
+</td>
+<td>
+   <p>
+   <!--
+   MatchConditionSubjectAccessReviewVersion specifies the SubjectAccessReview
+version the CEL expressions are evaluated against
+Valid values: v1
+Required, no default value
+   -->
+   matchConditionSubjectAccessReviewVersion 指定对 CEL 表达式求值时使用的 SubjectAccessReview 版本
```
Comment (tengqm) [inline content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md:1732]:
```suggestion
   matchConditionSubjectAccessReviewVersion 指定对 CEL 表达式求值时使用的 SubjectAccessReview 版本。
```

File: content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md [hunk 24]
```diff
@@ -471,3 +1563,343 @@ UDSTransport 设置通过 UDS 连接 konnectivity 服务器时需要的信息。
...
+   有效值：
+   </p>
+<ul>
+<!--
+<li>NoOpinion: continue to subsequent authorizers to see if one of
+them allows the request</li>
+<li>Deny: reject the request without consulting subsequent authorizers
+Required, with no default.</li>
+-->
+<li>NoOpinion：继续执行后续鉴权组件，看其中是否有组件允许该请求</li>
```
Comment (tengqm) [inline content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md:1759]:
```suggestion
<li>NoOpinion：继续执行后续鉴权组件，看其中是否有组件允许该请求；</li>
```

File: content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md [hunk 25]
```diff
@@ -471,3 +1563,343 @@ UDSTransport 设置通过 UDS 连接 konnectivity 服务器时需要的信息。
...
+<li>KubeConfigFile: use the file specified in kubeConfigFile to locate the
+server.</li>
+<li>InClusterConfig: use the in-cluster configuration to call the
+SubjectAccessReview API hosted by kube-apiserver. This mode is not
+allowed for kube-apiserver.</li>
+-->
+   <p>控制 Webhook 如何与服务器通信。
+有效值：</p>
+<ul>
+<li>KubeConfigFile：使用 kubeConfigFile 中指定的文件来定位服务器。</li>
```
Comment (tengqm) [inline content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md:1847]:
```suggestion
<li>KubeConfigFile：使用 <code>kubeConfigFile</code> 中指定的文件来定位服务器。</li>
```

File: content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md [hunk 26]
```diff
@@ -142,106 +259,1081 @@ TracingConfiguration 为跟踪客户端提供版本化的配置信息。
...
+对于身份验证配置，声明或前缀都没有默认值。声明和前缀必须显式设置。
+对于声明，如果在传统标志方法中未设置 --oidc-username-claim，
+请在身份验证配置中配置 username.claim=&quot;sub&quot;。
+对于前缀：
+(1) --oidc-username-prefix="-", 未添加前缀到用户名。要实现相同的行为，请在身份验证配置中设置
+username.prefix=&quot;&quot;
+(2) --oidc-username-prefix=&quot;&quot; 并且 --oidc-username-claim != &quot;email&quot;，
+前缀为 &quot;&lt;--oidc-issuer-url 的值&gt;#&quot;。要实现相同的行为，请在身份验证配置中设置
+username.prefix=&quot;&lt;issuer.url 的值&gt;#&quot;。
+(3) --oidc-username-prefix=&quot;。要实现相同的行为，请在身份验证配置中设置
```
[the remark this answers is not in the document]
Comment (mengjiao-liu) [reply content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md]: I missed this! Thanks for the catch!

**Pull request merged**